Sources/HTTPTypes/HTTPField.swift

```diff
@@ -20,28 +20,32 @@ public struct HTTPField: Sendable, Hashable {
     /// The strategy for whether the field is indexed in the HPACK or QPACK dynamic table.
     public struct DynamicTableIndexingStrategy: Sendable, Hashable {
         /// Default strategy.
+        @inlinable
         public static var automatic: Self { .init(uncheckedValue: 0) }
 
         /// Always put this field in the dynamic table if possible.
+        @inlinable
         public static var prefer: Self { .init(uncheckedValue: 1) }
 
         /// Don't put this field in the dynamic table.
+        @inlinable
         public static var avoid: Self { .init(uncheckedValue: 2) }
 
         /// Don't put this field in the dynamic table, and set a flag to disallow intermediaries to
         /// index this field.
+        @inlinable
         public static var disallow: Self { .init(uncheckedValue: 3) }
 
-        fileprivate let rawValue: UInt8
+        /* fileprivate but */ @usableFromInline let rawValue: UInt8
 
-        private static let maxRawValue: UInt8 = 3
+        /* private but */ @usableFromInline static let maxRawValue: UInt8 = 3
 
-        private init(uncheckedValue: UInt8) {
+        /* private but */ @inlinable init(uncheckedValue: UInt8) {
             assert(uncheckedValue <= Self.maxRawValue)
             self.rawValue = uncheckedValue
         }
 
-        fileprivate init?(rawValue: UInt8) {
+        /* fileprivate but */ @inlinable init?(rawValue: UInt8) {
```

Comment:
Is it worth renaming these (and elsewhere in this PR where we had to relax the access modifier) by prepending a `_` (in the case of the inits, maybe append to the argument name)?

Reply:
These are also going to run afoul of the somewhat-debated "no block comments" rule in our standard formatter configuration.